crates/pipeline_manager/migrations/V8__deployed_pipelines.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE deployed_pipeline(
+    id uuid PRIMARY KEY,
+    pipeline_id uuid NOT NULL,
+    tenant_id uuid NOT NULL,
+    config VARCHAR NOT NULL,
+    FOREIGN KEY (tenant_id) REFERENCES tenant(id) ON DELETE CASCADE,
+    CONSTRAINT unique_deployment_id UNIQUE (id, tenant_id),
+    CONSTRAINT unique_pipeline_id UNIQUE (pipeline_id, tenant_id)
+);
+
+-- TODO: migrate NULL pipeline-statuses and remove history tables
```

Comment:
Within the `pipeline` table, the field `last_revision` could be renamed to `latest_deployment`.

Reply:
We likely don't need this field anymore. I'm going to remove it through a migration.